_posts/2020-05-30-f-droid-repo.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Team NewPipe publishes own F-Droid repository"
+short: "NewPipe F-Droid repository"
+date: 2020-05-30 23:30:00 +0100
+categories: [announcement, f-droid, pinned]
+author: TheAssassin
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Team NewPipe proudly announces a self-hosted F-Droid repository, providing the official NewPipe app as well as -- in the future -- NewPipe Legacy. The F-Droid repository is a convenient way for users to update more quickly, which is especially useful whenever one of the supported services break.
+
+<!-- more -->
+
+## How to add the repository to F-Droid
+
+Instead of telling you the boring story behind the repository, let's first talk about how you can use it from F-Droid on your device(s).
+
+The easiest way to add the repository is to scan the QR code below or [click this link](fdroidrepo://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501) if you're viewing this on your device already. F-Droid should pop up on both, prompting you to add the repository. For the paranoid (*which is not a bad idea, especially in this context!*), you can verify the fingerprint F-Droid shows you with the one in the "copy-pastable" URL below.
+
+<center>
+    <img src="/img/fdroid-repo-qrcode.svg" alt="NewPipe F-Droid repository QR code" style="width: 250px">
+</center>
+
+Alternatively, in F-Droid, open the *Settings*, and choose *Repositories*. Then press the *plus* button, and paste the link if needed.
+
+Once you added the repository, update your package lists. If you had NewPipe installed from the official F-Droid repository before, you will have to uninstall this one before F-Droid will display the packages from our repository.
+
+The URL you need to copy is below. It includes the key's fingerprint for your convenience.
```

Comment:
Why is this placed down here? The instructions to add the repository two paragraphs above.

Reply:
The recommended way is to use the QR code. This is just a fallback.

Reply:
@TheAssassin: Above (line 20) is a clickable link, so imho there is no need for a text link.

Reply:
Less tech-savvy users might not understand how to copy a link. So having something in plain text they can copy is nice. Also, it allows for introspecting the link (on the browser, I can move the mouse over a link and wait a few milliseconds, but on a smartphone this doesn't work...).

Reply:
>Less tech-savvy users might not understand how to copy a link.

You don't need to copy it, just click it and it'll open the F-Droid app.

>on the browser, I can move the mouse over a link and wait a few milliseconds, but on a smartphone this doesn't work...

On a smartphone you could long press it.

Reply:
The line 20 link cannot be pasted into the repo box without editing out the `fdroidrepo://` part, which is why having the plain link without it is nice.